File: informacaoaluno.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;
import java.sql.Statement;

public class informacaoaluno {

    public static void main(String[] args) {
        // Daniele - Atualização de Informações de Alunos e Cursos

        // Dados de conexão com o banco de dados
        String url = "jdbc:mysql://seu_servidor/seu_banco_de_dados";
        String usuario = "root";
        String senha = "";

        try {
            // Estabelece a conexão
            Connection conexao = DriverManager.getConnection(url, usuario, senha);
            Statement stmt = conexao.createStatement();

            // Atualiza informações do aluno
            String sqlAluno = "UPDATE alunos SET nome='Julia Atualizada', turma='D' WHERE id_aluno=2";
            stmt.executeUpdate(sqlAluno);

            // Atualiza informações do curso associado ao aluno
            String sqlCurso = "UPDATE cursos SET nome_curso='Ciências Atualizado', instrutor='Professora Silva' WHERE id_curso=2";
            stmt.executeUpdate(sqlCurso);

            // Adiciona novo curso associado ao aluno
            String sqlNovoCurso = "INSERT INTO cursos (nome_curso, instrutor) VALUES ('História', 'Professor Oliveira')";
            stmt.executeUpdate(sqlNovoCurso);

            // Fecha a conexão
            conexao.close();

        } catch (SQLException e) {
            e.printStackTrace();
        }
    }
}

File: informacaoaluno.php
<?php // Tayna
// Dados de conexão com o banco de dados
$servername = "localhost";
$username = "root";
$password = "";
$dbname = "informacaoaluno";

$conn = new mysqli($servername, $username, $password, $dbname);

if ($conn->connect_error) {
    die("Falha na conexão: " . $conn->connect_error);
}

$sqlAluno = "UPDATE alunos SET nome='Lucas Atualizado', turma='C' WHERE id_aluno=1";
$conn->query($sqlAluno);

$sqlCurso = "UPDATE cursos SET nome_curso='Matemática Atualizado', instrutor='Professor Silva' WHERE id_curso=1";
$conn->query($sqlCurso);

$sqlNovoCurso = "INSERT INTO cursos (nome_curso, instrutor) VALUES ('Português', 'Professora Maria')";
$conn->query($sqlNovoCurso);

$conn->close();
?>
